src/test/fuzz/validation_load_mempool.cpp

```diff
@@ -36,9 +37,12 @@ FUZZ_TARGET_INIT(validation_load_mempool, initialize_validation_load_mempool)
 
     CTxMemPool pool{MemPoolOptionsForTest(g_setup->m_node)};
 
+    auto& chainstate{static_cast<DummyChainState&>(g_setup->m_node.chainman->ActiveChainstate())};
+    chainstate.SetMempool(&pool);
+
     auto fuzzed_fopen = [&](const fs::path&, const char*) {
         return fuzzed_file_provider.open();
     };
-    (void)LoadMempool(pool, g_setup->m_node.chainman->ActiveChainstate(), fuzzed_fopen);
+    (void)chainstate.LoadMempool(g_setup->m_args);
```

Comment:
c4249c1243981f42091950d1295c2be6bd4bf118: You are removing fuzzed_open. This will disable the fuzz tests. 


Also there is a typo in the commit message "it definitely is differece".

Reply:
Fixed in b857ac60d9